### DISCUSSION TITLE
El elemento recipients->vatIdType es obligatorio, qué se hace con los tickets simplificados?

### DISCUSSION THREAD

**Discussion by brahimbjz** [Barnetik/tbai-php-lib#52] [Q&A] [answered]
Estoy leyendo el archivo `docs/ticketbai-json.md` y veo que el elemento `recipients->vatIdType` es obligatorio y que requiere por lo tanto especificar el tipo de documento del cliente. ¿Qué se debería introducir en caso de ser una factura simplificada? Es decir, que no está vinculada a ningún cliente. Muchas gracias!

```
"vatIdType": {
    "type": "string",
    "enum": [
         "02",
         "03",
         "04",
         "05",
        "06"
    ],
    "default": "02",
    "description": "\nDokumentu mota - Tipo de documento:\n * 02: IFZ - NIF\n * 03: Pasaportea - Pasaporte\n * 04: Egoitza dagoen herrialdeak edo lurraldeak emandako nortasun agiri ofiziala - Documento oficial de identificaci\u00f3n expedido por el pa\u00eds o territorio de residencia\n * 05: Egoitza ziurtagiria - Certificado de residencia\n * 06: Beste frogagiri bat - Otro documento probatorio\n                    "
}
```

**Comment by arraintxo** [comment, marked as answer]
Egun on @brahimbjz ,

en el caso de las facturas simplificadas directamente puedes omitir el apartado de recipients.

Tienes un ejemplo de factura simplificada aquí:

https://github.com/Barnetik/tbai-php-lib/blob/main/tests/Barnetik/Tbai/__files/tbai-simplified-rectification-without-recipient-sample.json
